app/models/concerns/hyrax/ability.rb

```diff
@@ -93,13 +108,28 @@ def download_permissions
       end
     end
 
-    # Add this to your ability_logic if you want all logged in users to be able
-    # to submit content
+    ##
+    # @api public
+    #
+    # Add this to your {.ability_logic} if you want all logged in users to be able
+    # to submit content.
+    #
```

Comment:
I think a note should maybe be added here that while it isn't included in ability_logic by default by this file it IS included by default in the application's ability by the install generator.

Reply:
good call.

Reply:
Perfect.